test/helpers/util/Expect.js

```diff
@@ -1,9 +1,38 @@
 /*global JSON */
-define(['vendor/lib/q'], function (Q) {
+define([
+  'okta',
+  'vendor/lib/q',
+  'helpers/mocks/Util',
+  'sandbox'
+], function (Okta, Q, Util, $sandbox) {
 
   var fn = {};
   var PRIMARY_AUTH = 'primary-auth';
   var ENROLL_CHOICES = 'enroll-choices';
+  var $ = Okta.$;
+
+  fn.describe = function(desc, fn) {
+    return describe(desc, function() {
+
+      beforeAll(function () { /* jshint ignore:line */
```

Comment:
What is the ignore here for? If we need an ignore, we should ignore the specific error